{
  "uid" : "27fdf052853fbc2d",
  "name" : "test_delete_dmsg",
  "fullName" : "test.test_messages_functionality#test_delete_dmsg",
  "historyId" : "f19e9c8f1ee94d8815bdda0bfa138b0a",
  "time" : {
    "start" : 1712056717992,
    "stop" : 1712056717992,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: File target.json not found",
  "statusTrace" : "request = <SubRequest 'config' for <Function test_get_user_ban>>\n\n    @pytest.fixture(scope=\"session\", autouse=True)\n    def config(request):\n        if \"GITHUB_ACTIONS\" in os.environ and os.environ[\"GITHUB_ACTIONS\"] == \"true\":\n            # Возвращаем данные из переменных окружения\n            return {\n                \"web\": {\n                    \"baseUrl\": os.getenv(\"BASE_URL\")\n                },\n                \"api\": {\n                    \"baseUrl\": os.getenv(\"API_BASE_URL\"),\n                    \"ssoUrl\": os.getenv(\"SSO_URL\"),\n                    \"X-Node-Id\": os.getenv(\"X_NODE_ID\"),\n                    \"Login_player\": os.getenv(\"LOGIN_PLAYER\"),\n                    \"Password_player\": os.getenv(\"PASSWORD_PLAYER\"),\n                    \"Login_admin\": os.getenv(\"LOGIN_ADMIN\"),\n                    \"Password_admin\": os.getenv(\"PASSWORD_ADMIN\")\n                }\n            }\n        else:\n            # Проверяем наличие файла target.json\n            config_file = request.config.getoption(\"--target\")\n            if not os.path.exists(config_file):\n>               raise FileNotFoundError(f\"File {config_file} not found\")\nE               FileNotFoundError: File target.json not found\n\nconftest.py:31: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "config",
    "time" : {
      "start" : 1712056717419,
      "stop" : 1712056717419,
      "duration" : 0
    },
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: File target.json not found\n",
    "statusTrace" : "  File \"/usr/local/lib/python3.9/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.9/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/usr/local/lib/python3.9/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/usr/workspace/conftest.py\", line 31, in config\n    raise FileNotFoundError(f\"File {config_file} not found\")\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "critical"
  }, {
    "name" : "story",
    "value" : "Deleting Messages"
  }, {
    "name" : "feature",
    "value" : "Message Operations"
  }, {
    "name" : "parentSuite",
    "value" : "test"
  }, {
    "name" : "suite",
    "value" : "test_messages_functionality"
  }, {
    "name" : "host",
    "value" : "43e0a1e0dd95"
  }, {
    "name" : "thread",
    "value" : "1-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_messages_functionality"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "critical",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 4,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 8
      },
      "items" : [ {
        "uid" : "403a2519d7f31283",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: '/usr/workspace/target.json'",
        "time" : {
          "start" : 1712056501424,
          "stop" : 1712056501424,
          "duration" : 0
        }
      }, {
        "uid" : "eed387825774e686",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: '/usr/workspace/target.json'",
        "time" : {
          "start" : 1712055502731,
          "stop" : 1712055502731,
          "duration" : 0
        }
      }, {
        "uid" : "20adc2c17823a492",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: '/usr/workspace/target.json'",
        "time" : {
          "start" : 1712053563600,
          "stop" : 1712053563600,
          "duration" : 0
        }
      }, {
        "uid" : "efa3b868e8f5d5dd",
        "status" : "failed",
        "statusDetails" : "AssertionError: Response status code is not 200\nassert 504 == 200\n +  where 504 = <Response [504]>.status_code",
        "time" : {
          "start" : 1712052167631,
          "stop" : 1712052227823,
          "duration" : 60192
        }
      }, {
        "uid" : "c082b15ce9b0055b",
        "status" : "failed",
        "statusDetails" : "AssertionError: Response status code is not 200\nassert 504 == 200\n +  where 504 = <Response [504]>.status_code",
        "time" : {
          "start" : 1712049675647,
          "stop" : 1712049735770,
          "duration" : 60123
        }
      }, {
        "uid" : "54390499faacf349",
        "status" : "failed",
        "statusDetails" : "AssertionError: Response status code is not 200\nassert 504 == 200\n +  where 504 = <Response [504]>.status_code",
        "time" : {
          "start" : 1712042058529,
          "stop" : 1712042118665,
          "duration" : 60136
        }
      }, {
        "uid" : "e77292e55f23047f",
        "status" : "failed",
        "statusDetails" : "AssertionError: Response status code is not 200\nassert 504 == 200\n +  where 504 = <Response [504]>.status_code",
        "time" : {
          "start" : 1711622017683,
          "stop" : 1711622077825,
          "duration" : 60142
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "27fdf052853fbc2d.json",
  "parameterValues" : [ ]
}